src/components/Hero/Hero.module.scss

```diff
@@ -85,3 +85,49 @@
   height: 100%;
   background-color: rgba(0, 0, 0, 0.3);
 }
+
+@media (max-width: 600px) {
+  .heroFooter {
+    min-width: 100vw;
+    font-size: 20vw;
+    gap: 1vh;
+    right: 0;
+    flex-direction: column;
+    bottom: 14;
+    padding: 0 1vw 2vh 0;
```

Comment:
simpler usually to just have static values for paddings etc. If you need it adjusted for a specific viewport, I would just introduce another media query. That way we know what to expect for different devices. We usually do mobile, desktop, and tablet if needed